[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\sh372234\\AppData\\Local\\Temp\\1\\quarto-sessioneddb2b0beab8b5dc'\n"]
